src/view_element.cpp

```diff
@@ -1399,6 +1399,9 @@ void View::DrawSyl(DeviceContext *dc, LayerElement *element, Layer *layer, Staff
         dc->SetFont(m_doc->GetDrawingLyricFont(staff->m_drawingStaffSize));
     }
 
+    dc->GetFont()->SetStyle(syl->HasFontstyle() ? syl->GetFontstyle() : data_FONTSTYLE::FONTSTYLE_normal);
+    dc->GetFont()->SetWeight(syl->HasFontweight() ? syl->GetFontweight() : data_FONTWEIGHT::FONTWEIGHT_normal);
```

Comment:
No need to prefix `FONTSTYLE_normal` and `FONTWEIGHT_normal`
Again, do we always want / need the font style / weigh to be set to normal?

Reply:
Currently if style/weight is not specified in the input, we assume that it is normal. Normal means default. From rendering point of view a font type cant be undefined. And for sure we can omit these attributes in the output if they are _normal.